jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1,4 +1,5 @@
 #region Functions
+. "$PSScriptRoot\ProgressForm.ps1"
```

Comment:
Technically we do not need to load this when the program starts. I'd like you to come up with a more elegant way of selectively loading this series of functions. 

Completely related, this is only needed when we run the GUI version of the tool.